pkgs/build-support/build-dotnet-module/default.nix

```diff
@@ -21,11 +21,22 @@
 # Unfortunately, dotnet has no method for doing this automatically.
 # If unset, all executables in the projects root will get installed. This may cause bloat!
 , executables ? null
+# Pack module as nupkg and push it to `$out/share`. Disabled by default
```

Comment:
Nit: I wouldn't bother spelling out that this is disabled by default because that's obvious on the very next line.

Reply:
Removed in the updated commit